### ISSUE TITLE
[bug]: Unable to start 3.0.0a6 web UI: Python error in ImageRecordStorageBase get_most_recent_image_for_board()

### ISSUE THREAD

**Issue by panicsteve** [invoke-ai/InvokeAI#3632]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

macOS

### GPU

mps

### What version did you experience this issue on?

3.0.0a6

### What happened?

Environment: Mac Studio M1 Ultra / 64GB RAM / Ventura 13.4.1

1. Used install.sh to install 3.0.0a6 into a brand new empty directory.  It reported success.  I decided not to migrate anything from my 2.x.x installation.
2. Chose option 1 after running invoke.sh in the brand new 3.0.0a6 installation to launch the web UI
3. Received the following Python error Traceback:

```
Generate images with a browser-based interface
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ /Users/stevenf/Downloads/InvokeAI-3/.venv/bin/invokeai-web:5 in <module>                         │
│                                                                                                  │
│   2 # -*- coding: utf-8 -*-                                                                      │
│   3 import re                                                                                    │
│   4 import sys                                                                                   │
│ ❱ 5 from invokeai.app.api_app import invoke_api                                                  │
│   6 if __name__ == '__main__':                                                                   │
│   7 │   sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])                         │
│   8 │   sys.exit(invoke_api())                                                                   │
│                                                                                                  │
│ /Users/stevenf/Downloads/InvokeAI-3/.venv/lib/python3.9/site-packages/invokeai/app/api_app.py:26 │
│ in <module>                                                                                      │
│                                                                                                  │
│    23                                                                                            │
│    24 import invokeai.frontend.web as web_dir                                                    │
│    25                                                                                            │
│ ❱  26 from .api.dependencies import ApiDependencies                                              │
│    27 from .api.routers import sessions, models, images, boards, board_images                    │
│    28 from .api.sockets import SocketIO                                                          │
│    29 from .invocations.baseinvocation import BaseInvocation                                     │
│                                                                                                  │
│ /Users/stevenf/Downloads/InvokeAI-3/.venv/lib/python3.9/site-packages/invokeai/app/api/dependenc │
│ ies.py:5 in <module>                                                                             │
│                                                                                                  │
│     2                                                                                            │
│     3 from logging import Logger                                                                 │
│     4 import os                                                                                  │
│ ❱   5 from invokeai.app.services.board_image_record_storage import (                             │
│     6 │   SqliteBoardImageRecordStorage,                                                         │
│     7 )                                                                                          │
│     8 from invokeai.app.services.board_images import (                                           │
│                                                                                                  │
│ /Users/stevenf/Downloads/InvokeAI-3/.venv/lib/python3.9/site-packages/invokeai/app/services/boar │
│ d_image_record_storage.py:5 in <module>                                                          │
│                                                                                                  │
│     2 import sqlite3                                                                             │
│     3 import threading                                                                           │
│     4 from typing import Union, cast                                                             │
│ ❱   5 from invokeai.app.services.board_record_storage import BoardRecord                         │
│     6                                                                                            │
│     7 from invokeai.app.services.image_record_storage import OffsetPaginatedResults              │
│     8 from invokeai.app.services.models.image_record import (                                    │
│                                                                                                  │
│ /Users/stevenf/Downloads/InvokeAI-3/.venv/lib/python3.9/site-packages/invokeai/app/services/boar │
│ d_record_storage.py:7 in <module>                                                                │
│                                                                                                  │
│     4 import threading                                                                           │
│     5 from typing import Optional, Union                                                         │
│     6 import uuid                                                                                │
│ ❱   7 from invokeai.app.services.image_record_storage import OffsetPaginatedResults              │
│     8 from invokeai.app.services.models.board_record import (                                    │
│     9 │   BoardRecord,                                                                           │
│    10 │   deserialize_board_record,                                                              │
│                                                                                                  │
│ /Users/stevenf/Downloads/InvokeAI-3/.venv/lib/python3.9/site-packages/invokeai/app/services/imag │
│ e_record_storage.py:58 in <module>                                                               │
│                                                                                                  │
│    55 │   │   super().__init__(message)                                                          │
│    56                                                                                            │
│    57                                                                                            │
│ ❱  58 class ImageRecordStorageBase(ABC):                                                         │
│    59 │   """Low-level service responsible for interfacing with the image record store."""       │
│    60 │                                                                                          │
│    61 │   # TODO: Implement an `update()` method                                                 │
│                                                                                                  │
│ /Users/stevenf/Downloads/InvokeAI-3/.venv/lib/python3.9/site-packages/invokeai/app/services/imag │
│ e_record_storage.py:119 in ImageRecordStorageBase                                                │
│                                                                                                  │
│   116 │   │   pass                                                                               │
│   117 │                                                                                          │
│   118 │   @abstractmethod                                                                        │
│ ❱ 119 │   def get_most_recent_image_for_board(self, board_id: str) -> ImageRecord | None:        │
│   120 │   │   """Gets the most recent image for a board."""                                      │
│   121 │   │   pass                                                                               │
│   122                                                                                            │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
TypeError: unsupported operand type(s) for |: 'ModelMetaclass' and 'NoneType'
```

### Additional context

This is the same process I use to start up InvokeAI 2 on this computer (from its separate installation directory).

**Comment by panicsteve** [comment]
Python in the venv reports that it is 3.9.16, if that is helpful.

**Comment by StAlKeR7779** [comment]
Yeah, we already notice it. Fix in progress - #3630

**Comment by panicsteve** [comment]
This appears to have been resolved in 3.0 alpha 7 and 8.
